### PR TITLE
[WIP] Adding Time Signifier Component 

### PR DESCRIPTION
**Pull request by Schachte**
##  Background:
Alternative commercial applications such as Google Calendar have some sort of time signifier that delineates the users current time by showing a visual, red horizontal line across the calendar. I find this particularly useful to have a frame of reference of my day compared to the current time in a quick visual glance.  Referenced from issue #37 

#### Note:
Work in progress. Ill need to update the README for the prop usage and probably modify the tests to pass + linter to pass. Just thought this would be a useful feature that people could opt-in to. 

### Screenshots:

[GOOGLE]
image

[REACT AVAILABLE TIMES]
image

### CODE REVIEW

File: src/Week.jsx [hunk 1]
```diff
@@ -137,6 +137,16 @@ export default class Week extends PureComponent {
     return result;
   }
 
+  renderCurrentTimeSignifier(timezone) {
+    const currentEpoch = moment().tz(timezone).format("hh:mm")
```
Comment (Schachte) [inline src/Week.jsx:141]: Change this variable from currentEpoch -> formattedTime

File: src/Week.jsx [hunk 2]
```diff
@@ -194,26 +206,30 @@ export default class Week extends PureComponent {
...
-                key={day.date}
-                date={day.date}
-                events={dayEvents[i]}
-                initialSelections={daySelections[i]}
-                onChange={this.handleDayChange}
-                hourLimits={this.generateHourLimits()}
-                touchToDeleteSelection={touchToDeleteSelection}
-              />
+              <div>
+              {timeSignifier && this.renderCurrentTimeSignifier(timeZone)}
```
Comment (trotzig) [inline src/Week.jsx:217]: If you move this below the <Day>, can you remove the 999999 z-index? I prefer relying on stacking contexts and DOM order to render things correctly. If that doesn't work, it's still fine with the z-index, but I don't think you need 999999 (2 should be enough).

**Review by trotzig — APPROVED** [review]
Looks good to me! The build failure isn't your fault, you can ignore that. 

I'll let you decide if you want to make adjustments. When you are done, let me know and I'll merge and release.

File: src/Week.jsx (hunk 2 above)
Comment (Schachte) [reply src/Week.jsx:217]: Great, will do thanks!

**Pull request closed without merging**